tests/image-stream/utils.spec.js

```diff
@@ -26,44 +26,34 @@ describe('ImageStream: Utils', function() {
 
   describe('.getArchiveMimeType()', function() {
 
-    it('should resolve application/x-bzip2 for a bz2 archive', function(done) {
+    it('should resolve application/x-bzip2 for a bz2 archive', function() {
```

Comment:
If .getArchiveMimeType is now back to Promises, does this need the `done` that the original version had?